src/mode/terminal.rs

```diff
@@ -170,14 +170,15 @@ where
 
         let numchars = match display_size {
             DisplaySize::Display128x64 => 128,
+            DisplaySize::Display132x64 => 64,
```

Comment:
I copied 64 from 128x32 because the display is _really_ 128x32

Reply:
I'm not following this comment. Shouldn't the above line be 32?